snmp/tests/test_profiles.py

```diff
@@ -2911,3 +2910,92 @@ def _check_common_asr(aggregator, tags):
 
     for metric in RATE_METRICS:
         aggregator.assert_metric('snmp.{}'.format(metric), metric_type=aggregator.RATE, tags=tags)
+
+
+@pytest.mark.usefixtures("dd_environment")
+def test_cisco_legacy_wlc(aggregator):
+    run_profile_check('cisco-5500-wlc', profile_name='cisco-legacy-wlc')
+    common_tags = common.CHECK_TAGS + ['snmp_profile:cisco-legacy-wlc', 'device_vendor:cisco', 'snmp_host:DVA00WLC01']
```

Comment:
the snmp_host value here needs to be updated with the new one (this is why [these tests](https://github.com/DataDog/integrations-core/pull/14175/checks?check_run_id=12047437594) are failing) 
```suggestion
    common_tags = common.CHECK_TAGS + ['snmp_profile:cisco-legacy-wlc', 'device_vendor:cisco', 'snmp_host:DDOGWLC']
```